packages/build/src/utils/blobs.ts

```diff
@@ -84,29 +101,49 @@ export const scanForBlobs = async function (buildDir: string, packagePath?: stri
 const METADATA_PREFIX = '$'
 const METADATA_SUFFIX = '.json'
 
-/** Given output directory, find all file paths to upload excluding metadata files */
-export const getKeysToUpload = async (blobsDir: string): Promise<string[]> => {
+/**
+ * Returns the blobs that should be uploaded for a given directory tree. The
+ * result is an array with the blob key, the path to its data file, and the
+ * path to its metadata file.
+ */
+export const getKeysToUpload = async (blobsDir: string) => {
```

Comment:
This is the old method for the legacy APIs (file-based uploads and Deploy Configuration API). We're just changing the shape of the return value to be an array of objects.